node/src/Pos/Launcher/Param.hs

```diff
@@ -55,23 +56,24 @@ data TransportParams = TransportParams
 -- | This data type contains all data necessary to launch node and
 -- known in advance (from CLI, configs, etc.)
 data NodeParams = NodeParams
-    { npDbPathM        :: !FilePath             -- ^ Path to node's database
-    , npRebuildDb      :: !Bool                 -- ^ @True@ if data-base should be rebuilt
-    , npSystemStart    :: !Timestamp            -- ^ System start
-    , npSecretKey      :: !SecretKey            -- ^ Primary secret key of node
-    , npUserSecret     :: !UserSecret           -- ^ All node secret keys
-    , npBaseParams     :: !BaseParams           -- ^ See 'BaseParams'
-    , npGenesisCtx     :: !GenesisContext       -- ^ Predefined genesis context
-    , npJLFile         :: !(Maybe FilePath)     -- TODO COMMENT
-    , npReportServers  :: ![Text]               -- ^ List of report server URLs
-    , npUpdateParams   :: !UpdateParams         -- ^ Params for update system
-    , npUseNTP         :: !Bool                 -- ^ Whether to use synchronisation with NTP servers.
-    , npTransport      :: !TransportParams      -- ^ (TCP) transport parameters.
-    , npEnableMetrics  :: !Bool                 -- ^ Gather runtime statistics.
-    , npEkgParams      :: !(Maybe EkgParams)    -- ^ EKG statistics monitoring.
-    , npStatsdParams   :: !(Maybe StatsdParams) -- ^ statsd statistics backend.
+    { npDbPathM        :: !FilePath                         -- ^ Path to node's database
+    , npRebuildDb      :: !Bool                             -- ^ @True@ if data-base should be rebuilt
+    , npSystemStart    :: !Timestamp                        -- ^ System start
+    , npSecretKey      :: !SecretKey                        -- ^ Primary secret key of node
+    , npUserSecret     :: !UserSecret                       -- ^ All node secret keys
+    , npBaseParams     :: !BaseParams                       -- ^ See 'BaseParams'
+    , npGenesisCtx     :: !GenesisContext                   -- ^ Predefined genesis context
+    , npJLFile         :: !(Maybe FilePath)                 -- TODO COMMENT
+    , npReportServers  :: ![Text]                           -- ^ List of report server URLs
+    , npUpdateParams   :: !UpdateParams                     -- ^ Params for update system
+    , npUseNTP         :: !Bool                             -- ^ Whether to use synchronisation with NTP servers.
+    , npTransport      :: !TransportParams                  -- ^ (TCP) transport parameters.
+    , npRoute53Params  :: !(Maybe Route53HealthCheckParams) -- ^ Where to listen for the Route53 DNS health-check.
```

Comment:
If this were just `NetworkAddress` we also wouldn't need to change all the layout :-)